src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -192,6 +155,9 @@ private Task WriteBatchBytes(byte[] batchBytes)
             });
         }
 
+        public void DispatchBufferedRenders()
```

Comment:
The naming of this seems odd to me. From the name, I thought the renders themselves (as in, the outputs from rendering) were being buffered, but really it's the *instructions to render* that are buffered.

It would seem clearer to me if this were simply `ProcessRenderQueue`.